### DISCUSSION TITLE
我们无法从开源本身获利 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#92] [Announcements]
# 我们无法从开源本身获利 - 「开源之道」

https://opensourceway.community/posts/opensource/you_cant_make_money_from_open_source/

**Comment by ZoomQuiet** [comment]
> 是也乎,(￣▽￣)

标题党, 关键是 利 如何定义了;

类似上流战略, 那么占到控制优势就已经是收益了,
只是这类无法简单兑换成 金钱 的东西, 
对于一般企业财务系统来说, 根本没有能力以及心力去主动完成兑换...

PS:

原文链接已经丢失, 感觉是很久之前的文章...

**Comment by zhao-hai-ling** [comment]
赞

**Comment by zhao-hai-ling** [comment]
1.要论述：开源的商业化是否是成功的，不能仅仅依据红帽的商业化是否可复制来判断，事实上，开源商业化已经发展出许多种模式了，都走向了盈利。
2.为什么红帽的商业化不能复制，就像是从自然水转向瓶装水，市面是有很多个售卖的品牌，不止农夫山泉一家。

**Reply by lijiangsheng1** [reply]
答： 1、 这是一篇2016年的文章，2016年的云计算还没有今天这么发达，大家都在探索中～ 今天有了RedHat之外的更多模式，尤其是2018年的开源被资本青睐的一年之后，所以有很多的变化。
2、 我理解作者不是RedHat不能复制，而是说这样的模式只有在操作系统和基础设施的垂直领域内只能有一个领头羊，而很难在技术之外复制这样的模式，比如中间件、应用软件等。但是话说回来，想复制RedHat的有很多，但是都没有获得市场，比如暴力抄袭的Oracle（《开放式组织》里提到的RedHat危机），更多详情可参考我最近写的文章[开源软件供应链之道](https://www.opensourceway.community/posts/supply_chain/try-to-explain-open-source-supply-chain/)
